contrib/gitian-descriptors/gitian-win.yml

```diff
@@ -156,7 +156,7 @@ script: |
   done
 
   mkdir -p ${OUTDIR}/src
-  git archive --output=${OUTDIR}/src/${DISTNAME}.tar.gz HEAD
+  git archive --output=${OUTDIR}/src/${DISTNAME}.zip HEAD
```

Comment:
Why?

The tar is provided so that users can compile from source (without having to clone from GitHub and checkout the tag). I don't think we really support compiling on Windows. And when we do, the extra dependency to extract a tar file is probably not too much to ask for from a windows user.

Finally, as pointed out by @laanwj, this probably does need adjustment on the Website. Not sure if it is worth it.

Reply:
Commit dropped (https://github.com/bitcoin/bitcoin/pull/18331#issuecomment-599098374).